doc/source/mergingdata.rst

```diff
@@ -96,3 +96,25 @@ The `how` argument specifies the type of join that will occur and which geometry
 * ``inner``: use intersection of index values from both geodataframes; retain only the `left_df` geometry column
 
 Note more complicated spatial relationships can be studied by combining geometric operations with spatial join. To find all polygons within a given distance of a point, for example, one can first use the ``buffer`` method to expand each point into a circle of appropriate radius, then intersect those buffered circles with the polygons in question.
+
+
+Sjoin Performance
+~~~~~~~~~~~~~~~~~~
+
+When rtree is installed, ``sjoin`` attempts to re-use an existing spatial index on either `left_df` or `right_df`. Performance of multiple sjoins in a row involving a common GeoDataFrame may be improved by pre-generating the spatial index of the common GeoDataFrame prior to performing sjoins using ``df1.sindex()``.
+
+.. ipython:: python
```

Comment:
```suggestion
.. code-block:: python
```

If you are doing an illustration of code, and not an actual runnable code example, need to use code-block, as the ipython block gets actually ran during the doc build (to generate the output to show in the online docs)

Reply:
Oh neat, thanks! This is my first time trying to use .rst, so I was just copying off of nearby examples :)